client/src/cordova/apple/OutlineLib/VpnExtension/Sources/PacketTunnelProvider.m

```diff
@@ -49,11 +49,7 @@ @implementation PacketTunnelProvider
 
 - (id)init {
   self = [super init];
-#if (TARGET_OS_OSX || TARGET_OS_MACCATALYST)
```

Comment:
I came up with a better way to fix the issue. Instead of using different app groups for macos and ios, let's use the same one. That simplified the project.

We also don't need to stick to the old app group id. We can use a new one! that's because there's no important data in the app group that needs to be persisted long term. We use it to allow all the exectuables  (main, launcher, extension) to share logs, and for storing temporary preferences for vpn restarts.

So I replaced our app groups with group.org.getoutline.client.

This was a [helpful reference](https://forums.developer.apple.com/forums/thread/718126) clarifying that it can be anything on macOS, and should be "owned" if we use the app store. I added that app group to our account.

Reply:
I agree that's simpler, though I remember having to be careful with the bundle identifiers on Catalyst to ensure users wouldn't lose their servers as they were stored under a different identifier: https://github.com/Jigsaw-Code/outline-apps/blob/95941aba95ccc9523f02b9f2b6e178b874928bb3/client/src/cordova/plugin/apple/src/OutlinePlugin.swift#L332.

Have you confirmed the servers stored in local storage can be retrieved fine with the new bundle IDs on iOS and macOS?

Reply:
Oh sorry I just realized you are talking about app group. Yeah that looks fine. I never got that to stick because Xcode kept changing the app group from underneath my changes.

Reply:
I had the same thought, but the server storage is under the bundle identifier, not the app group. Running the app retrieved the list of servers fine on both ios and macos.

Those identifiers have been very confusing, and I'm glad we can clean them up!